build.sc

```diff
@@ -555,11 +555,8 @@ trait VectorSubsystem
         "--split-verilog",
         "--strip-debug-info",
         "--preserve-values=none",
-        "--preserve-aggregate=all",
         "--disable-annotation-unknown",
-        "--output-annotation-file=mfc.anno.json",
-        "--repl-seq-mem",
-        "--repl-seq-mem-file=repl-seq-mem.txt"
+        "--output-annotation-file=mfc.anno.json"
```

Comment:
This should be fine, since the subsystem is only meant to be subsystem test only(with BFM or FPGA).